File: scala-server/server/src/main/scala/com/teletracker/service/logging/TeletrackerLogsEnhancer.java
package com.teletracker.service.logging;

import com.google.cloud.logging.LogEntry;
import com.google.cloud.logging.LoggingEnhancer;

class TeletrackerLogsEnhancer implements LoggingEnhancer {
  public void enhanceLogEntry(LogEntry.Builder logEntry) {
    logEntry.addLabel("application", "teletracker-server");
  }
}

File: backend/server/src/main/scala/com/teletracker/service/logging/TeletrackerLogsEnhancer.scala
//package com.teletracker.service.logging;
//
//import com.google.cloud.logging.LogEntry
//import com.google.cloud.logging.LoggingEnhancer
//
//class TeletrackerLogsEnhancer extends LoggingEnhancer {
//  final private val application =
//    Option(System.getenv("APPLICATION_NAME")).getOrElse("teletracker-unknown")
//
//  def enhanceLogEntry(logEntry: LogEntry.Builder): Unit = {
//    logEntry.addLabel("application", application);
//  }
//}
